website/source/docs/commands/version.html.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: "docs"
+page_title: "version - Command"
+sidebar_title: "<code>version</code>"
+sidebar_current: "docs-commands-version"
+description: |-
+  The "version" command prints the version of Vault.
+---
+
+# version
+
+The `version` command prints the Vault version:
+
+```
+$ vault version
+Vault v1.2.3
+```
+
+It can also be printed by using the flags `--version` or `-v` to the `vault` command:
```

Comment:
Should "using" be "adding"?  Alternatively, "to" could be changed to "with" if we keep "using"

Reply:
Thanks, will go with "adding"!